### PR TITLE
Fix index name in to_data_frame()'s docstring

### LINKED ISSUE #11456: Spectrum's to_data_frame() wrong name for appended column
### Proposed documentation enhancement

The method's docstring refers to the appended column as "frequencies", but the returned dataframe has "freq".

### PR DESCRIPTION
**Pull request by ZviBaratz**
Changed "frequencies" to "freq", resolves #11456.

### CODE REVIEW

**Review by mmagnuski — APPROVED** [review]
Fine by me. Thanks @ZviBaratz.

**Comment by larsoner** [comment]
Thanks @ZviBaratz -- could you add a line to `doc/changes/latest.inc` mentioning this docstring improvement (maybe under the Bug section?) and add your name + URL to `doc/changes/names.inc`? Then we can give you proper credit for your change

**Comment by ZviBaratz** [comment]
Done. Thank you!

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -43,6 +43,7 @@ Bugs
 - Fix :func:`mne.io.read_raw_edf` when reading EDF data with different sampling rates and a mix of data channels when using ``infer_types=True`` (:gh:`11427` by `Alex Gramfort`_)
 - Fix how :class:`mne.channels.DigMontage` is set when using :func:`mne.preprocessing.ieeg.project_sensors_onto_brain` so that :func:`mne.Info.get_montage` works and does not return ``None`` (:gh:`11436` by `Alex Rockhill`_)
 - Fix configuration folder discovery on Windows, which would fail in certain edge cases; and produce a helpful error message if discovery still fails (:gh:`11441` by `Richard Höchenberger`_)
+- Fix :func:`mne.time_frequency.spectrum.to_data_frame`'s docstring to reflect the correct name for the appended frequencies column.
```
Comment (drammock) [inline doc/changes/latest.inc:46]:
```suggestion
- Fix :func:`mne.time_frequency.spectrum.to_data_frame`'s docstring to reflect the correct name for the appended frequencies column (:gh:`11457` by :newcontrib:`Zvi Baratz`)
```

Comment (ZviBaratz) [reply doc/changes/latest.inc]: Whoops, my bad. Thank you, @drammock.

Comment (drammock) [reply doc/changes/latest.inc]: no worries!

File: doc/changes/latest.inc [hunk 2]
```diff
@@ -56,6 +56,7 @@ Bugs
 - Fix :func:`mne.io.read_raw_edf` when reading EDF data with different sampling rates and a mix of data channels when using ``infer_types=True`` (:gh:`11427` by `Alex Gramfort`_)
 - Fix how :class:`mne.channels.DigMontage` is set when using :func:`mne.preprocessing.ieeg.project_sensors_onto_brain` so that :func:`mne.Info.get_montage` works and does not return ``None`` (:gh:`11436` by `Alex Rockhill`_)
 - Fix configuration folder discovery on Windows, which would fail in certain edge cases; and produce a helpful error message if discovery still fails (:gh:`11441` by `Richard Höchenberger`_)
+- Fix :func:`mne.time_frequency.spectrum.to_data_frame`'s docstring to reflect the correct name for the appended frequencies column (:gh:`11457` by :newcontrib:`Zvi Baratz`)
```
Comment (drammock) [inline doc/changes/latest.inc:59]:
```suggestion
- Fix :meth:`mne.time_frequency.Spectrum.to_data_frame`'s docstring to reflect the correct name for the appended frequencies column (:gh:`11457` by :newcontrib:`Zvi Baratz`)
```

**Pull request merged**